infra/modules/worklytics-psoxy-connection-generic/main.tf

```diff
@@ -22,6 +34,28 @@ Complete the following steps in Worklytics AFTER you have deployed the Psoxy ins
   5. Select `${var.psoxy_host_platform_id}` for "Proxy Instance Type".${local.per_setting_instructions_text}
   ${local.last_step}. Review any additional settings that connector supports, adjusting values as you see fit, then
      click "Connect".
+EOT
+
+
+  deep_link_instructions = <<EOT
+  1. Ensure you're authenticated with Worklytics. Either sign-in at `https://app.worklytics.co` with
```

Comment:
We should have a clear pattern for this: always link to central, deep linking feature should persist path and params

Reply:
yeah, you're right; I'll take this as a tech-debt task. In practice, customers no longer care about EU if they're proxy customer though.  Although if we have large EU-based customer they might.  Since the proxy, we no longer seem to get the case from customers where even US based customers are interested in EU for GDPR compliance purposes; people now recognize it doesn't really matter, and that proxy does more towards GDPR compliance than EU hosting would.

Reply:
(as an aside, I think we can drop EU support; but there's a strong case to keep ability to have central auth and then redirect people out a Google project specific to them. eg, instead of `app-eu.worklytics.co`, we'd have `acme.worklytics.co` and we could put that project wherever they want, and the data/processing/etc would all be totally segregated by customer.  A lot of overhead to do that atm, but in future it could make sense if a customer is big enough that the efficiencies of sharing some resources is less than benefits of segregation.  A real GCP project for production use has a fixed base cost of ~$1500/m though, once we activate all the security stuff it requires - so not really efficient to do unless customer really values it.)